config: fix SSL detection

The @WEBSOCK_HAVE_SSL@ is not replaced at configure time, and even if
it was, it would be replaced by an empty string if openssl is disabled,
thus still defining WEBSOCK_HAVE_SSL when we would not want it.

Instead, rely on config.h, which is properly generated by ./configure,
to provide the information about whether openssl is enabled or not.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>

diff -durN a/src/websock_config.h.in b/src/websock_config.h.in
--- a/src/websock_config.h.in
+++ b/src/websock_config.h.in
@@ -1,9 +1,10 @@
 #ifndef WEBSOCK_CONFIG_H
 #define WEBSOCK_CONFIG_H 1
 
+#include "config.h"
+
 #define WEBSOCK_PACKAGE_STRING @WEBSOCK_PACKAGE_STRING@
 #define WEBSOCK_PACKAGE_VERSION @WEBSOCK_PACKAGE_VERSION@
 #define WEBSOCK_PACKAGE_NAME @WEBSOCK_PACKAGE_NAME@
-#define WEBSOCK_HAVE_SSL @WEBSOCK_HAVE_SSL@
 
 #endif
